docs/gathered-data.md

```diff
@@ -159,37 +159,40 @@ Location in archive: config/configmaps/
 See: docs/insights-archive-sample/config/configmaps
 
 
+## ContainerImages
+
+collects essential information about running containers.
+Specifically, the age of pods, the set of running images and the container names are collected.
+
+Location in archive: config/running_containers.json
+
+
 ## MostRecentMetrics
 
 gathers cluster Federated Monitoring metrics.
 
 The GET REST query to URL /federate
 Gathered metrics:
-  ALERTS
   etcd_object_counts
   cluster_installer
   namespace CPU and memory usage
+  followed by at most 1137 lines of ALERTS metric
```

Comment:
Needs update after changing the limit (ideally together with the commit message)

Reply:
Good point; done.